docker/Dockerfile

```diff
@@ -19,7 +22,7 @@ COPY . /srv/app
 WORKDIR /srv/app
 RUN gem install bundler --version 2.1.4
 RUN bundle --version
-RUN bundle check || bundle install --full-index --without development -j $(nproc)
+RUN bundle install --path vendor/bundle --full-index --without development -j $(nproc)
```

Comment:
is it possible to add `--without development test` or not? my question is in case travis CI uses this image

Reply:
Yes this image is used by Travis to test. But I think we should use a variable allowing use to remove test gems for production image.

Like:
```dockerfile
RUN bundle install --path vendor/bundle --full-index --without ${WITHOUT} -j $(nproc)
```

where in Travis `WITHOUT=development` and for production image `WITHOUT="development test"`

What do you thing about this option ?

Reply:
if `APP_ENV` is already defined before we arrive here, we can condition on `APP_ENV=='test'` like `$([[ $APP_ENV = 'test' ]] && echo 'development' || echo 'development test')` without an extra env variable. this is bash syntax, I don't know the equivalent in Dockerfile syntax.

Reply:
In Dockerfile we use argument for this kind of value as it is only use for build. The commit contains now the default value with is *"development test"* allowing the production build not to change anything and in travis build, I added a `--build-arg WITHOUT=development`